test/python/algorithms/optimizers/test_spsa.py

```diff
@@ -255,3 +255,27 @@ def objective(x):
         with self.subTest("check number of function calls"):
             expected_nfev = 8  # 7 * maxiter + 1
             self.assertEqual(result.nfev, expected_nfev)
+
+    def test_point_sample(self):
+        """Test point sample function in QNSPSA"""
+
+        def fidelity(x, y):
```

Comment:
pylint's moaning about the unused `y` argument. Given this is just a stub function, you can make it shut up by giving the second argument a name starting with an underscore, like `_y` - that means "explicitly ignored".

Reply:
done, thanks!

Reply:
Oh man, and now Pylint's complaining that that's not a valid name...  I'm not a fan of these pylint rules at all.

At any rate, you can put a comment just under the docstring of `test_point_sample` that says `# pylint: disable=invalid-name` or `# pylint: disable=unused-argument` as appropriate (depending on which name for the argument you prefer using).